app/assets/stylesheets/layout/_layout.scss

```diff
@@ -45,6 +45,17 @@ body {
       margin-bottom: 40px;
     }
   }
+
+}
+
+.registrations .box, .sessions .box{
+  width: 40%!important;
+  margin: 0 auto;
+
+  form input[type="submit"]{
+    width: 100%;
+    margin: 10px 0 20px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 0 20px` instead of `10px 0 20px 0`